test/test_pod_helpers.rb

```diff
@@ -71,6 +71,12 @@ def test_new_architecture_enabled?
     refute(new_architecture_enabled?({}, v(0, 70, 999)))
     assert(new_architecture_enabled?({}, available_version))
 
+    # `RCT_NEW_ARCH_ENABLED` disables everything
+    ENV['RCT_NEW_ARCH_ENABLED'] = '0'
+
+    refute(new_architecture_enabled?({}, v(0, 70, 999)))
+    refute(new_architecture_enabled?({}, available_version))
```

Comment:
Can you also add the case where `fabric_enabled: true`:
```suggestion
    refute(new_architecture_enabled?({}, available_version))
    refute(new_architecture_enabled?({ fabric_enabled: true }, available_version))
```

Reply:
I added that and another for `turbomodule_enabled` for completeness.